test/tools/ossfuzz/yulProto.proto

```diff
@@ -21,6 +21,10 @@ message VarDecl {
   required Expression expr = 1;
 }
 
+message MultiVarDecl {
+  required uint32 num_vars = 1;
```

Comment:
Is this a default that can be changed? A comment in `protoToYul.cpp` mentions it supports max 4 vars

Reply:
The way this works is that, the fuzzer generates a value for `num_vars` that is in the range of `0--UINT32MAX`.

Inside the conversion function in the file `protoToYul.cpp`, we convert this random number to something between 2 and 4 (inclusive) using an offset and modulo operator.